src/index.js

```diff
@@ -0,0 +1,78 @@
+import './style.css';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
[OPTIONAL] Please make sure to fix linter errors without disabling linters so kindly fix liner error and remove this line from your codebase :slightly_smiling_face:

Reply:
Thank you very much